java/junior-book-guide.md

```diff
@@ -0,0 +1,17 @@
+## These books are recommended from junior to early mid level times.
+## Reading order is according to my personal experience, feel free to get out of the box.
+
```

Comment:
Let's follow some conventions regarding the formats of the information added.
Something like next: 
Let's have three files for according  categories for books: 
- Entry-Level-Books.md
- Mid-Level-Books.md
- Senior-Level-Books.md
and inside files itself let's have format like next:
Link: {link to resource}
Description: {Small description from you XP}
So basically everything in the file you wrote is ok, just reformat them to match common conventions.